modules/configuration/test/TestPlan.md

```diff
@@ -72,3 +72,25 @@ project_cohort_rel table in the database.
   [Manual Test]
 5. Test the breadcrumb to see if you can navigate back to the main config page.
   [Automation Test]
+6. Go back to the project configuration page and click on the subproject link and 
+confirm that it brings you to the subproject configuration page.
+  [Manual Test]
+
+## Diagnosis Evolution 
+
+1. Click on the diagnosis evolution link at the top of the page and confirm that it brings you to
+the diagnosis trajectory configuration page.
+  [Manual Test]
+2. Click the 'New Diagnosis Trajectory' tab in the left column navigation. Try to add a new trajectory.
+  [Manual Test]
+3. Test that you cannot add a diagnosis trajectory for any source field or instrument that does not exist in the selected visit. Also test that you can add multiple source fields.
```

Comment:
having an empty "Source Field" field (when you dont actually click on the "Add Field" button) gives a 409 "Source Field does not exist in instrument" error when the form should handle its own errors for required fields not filled out. happens for the trajectory name and order number fields as well

<img width="1073" alt="Screen Shot 2022-12-19 at 1 21 44 PM" src="https://user-images.githubusercontent.com/17415878/208493659-a440007c-49ce-4f06-9324-aaff99f50fbb.png">